e2e/test-component/scenarios/embedding-sdk/interactive-question.cy.spec.tsx

```diff
@@ -380,4 +381,33 @@ describe("scenarios > embedding-sdk > interactive-question", () => {
       });
     });
   });
+
+  it("should select sensible display for new questions (EMB-308)", () => {
+    mountSdkContent(<InteractiveQuestion questionId="new" />);
+    cy.log("Select data");
+    H.popover().findByRole("link", { name: "Orders" }).click();
+
+    cy.log("Select summarization");
```

Comment:
so unfortunately, there's not a good way to test `InteractiveAdHocQuestion` directly in E2E because we can only test public components in E2E since we import from `@metabase/embedding-sdk-react`.

This would work for now, until we have a better way to not click UI.